data/pokemon.json

```diff
@@ -164,14 +181,17 @@
     ]
   },
   {
-    "name": "nidoran♀"
+    "name": "nidoran♀",
+    "shiny": true
   },
   {
-    "name": "nidorina"
+    "name": "nidorina",
+    "shiny": true
   },
   {
     "name": "nidoqueen",
-    "tier": 4
+    "tier": 4,
+    "shiny": true
```

Comment:
Indeed might be possible as a shiny (since they're one of the known instances of evolved form can be found shiny in the wild)